Accessibility/Accessibility/AppDelegate.swift

```diff
@@ -11,11 +11,19 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-    var window: UIWindow?
+    var window:UIWindow? = UIWindow(frame: UIScreen.main.bounds)
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        
+        let mainVC = HomeShopVC()
+        let navController = UINavigationController(rootViewController: mainVC)
+        
+        //self.window? = UIWindow(frame: UIScreen.main.bounds)
+        self.window?.rootViewController = navController
+        self.window?.makeKeyAndVisible()
```

Comment:
Great job with all of this in the `AppDelegate`! I always forget about this initial setup when creating a new project.

Reply:
😃 thanks!